Anytype/Sources/PresentationLayer/TextEditor/Set/Views/Collection/SetCollectionView.swift

```diff
@@ -1,4 +1,5 @@
 import SwiftUI
+import Services
```

Comment:
is it possible to not disclose services in view layer?

Reply:
Import was removed due to SubscriptionId deletion here and below